setup.cfg

```diff
@@ -58,6 +58,7 @@ install_requires =
   multidict >=4.5, < 7.0
   propcache >= 0.2.0
   yarl >= 1.17.0, < 2.0
+  orjson >= 3.8.0 ; platform_python_implementation == "CPython"
```

Comment:
Adding such a mandatory dep to the metadata is probably harmful. Also, the change note is lying about it being optional.